flink-core/src/test/java/org/apache/flink/api/common/typeutils/base/array/CharPrimitiveArraySerializerTest.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.flink.api.common.typeutils.TypeSerializer;
 
 /** A test for the {@link LongPrimitiveArraySerializer}. */
```

Comment:
Probably not related to the main PR's goal however shouldn't it be
```suggestion
/** A test for the {@link CharPrimitiveArraySerializer}. */
```